cirq-core/cirq/transformers/analytical_decompositions/quantum_shannon_decomposition_test.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cirq import NamedQubit, Circuit, approx_eq
+from cirq.ops.common_gates import Rz, Ry, ZPowGate, CXPowGate
```

Comment:
Prefer to use `import cirq` followed by `cirq.<object>` for all the imported objects here. They all exist in global cirq namespace.

Reply:
Done